### ISSUE TITLE
Remove WollokNativeLobby

### ISSUE THREAD

**Issue by javierfernandes** [uqbar-project/wollok#498]
The native lobby is the "this" object of a wollok program. As programs are special constructions.
We should probably need to model de **wollok.lang.Program** class and then change the interpreter so that when it finds a WProgram it will instantiate that class. In that way it will be just treated as a regular object.

This will simplify the intepreter EvaluationContext model.

**Comment by npasserini** [comment]
Why a class and not an object

**Comment by javierfernandes** [comment]
Yes, it could be. I didn't think about it much. Just arosed while
documenting.
With object you mean a regular anonymous object and not a predefined wko
right?
I thought about having a class to have there documented any behavior we
want a program to have.
But i think that changing it to an object is the simplest change that will
simplify de interpreter quickly

**Comment by npasserini** [comment]
Nope, I was thinking that program could be just a wko, but maybe I'm wrong.

**Comment by javierfernandes** [comment]
Yeah, just that we don't want it to be accesible from anywhere.

Actually WKO's are just the same as objectliterals just that they are added
to the global scope. So it's the same

**Comment by npasserini** [comment]
Ok, right. Also I was thinking that you might have multiple programs in the
same project, so if you tried to make them wkos you might produce a
colision.

So, I agree it should be a class.

**Comment by javierfernandes** [comment]
But program have a name. So we could treat them as WKOs in terms of cross-references.
So maybe the grammar will also change.

Example

```
program pepitaFlights {
}
```

Program class
```xtend
class Program {
}
```

Another file used by the program

```
pepita.
```

**Comment by javierfernandes** [comment]
This will also solve the program with "weird" stack trace element for programs

```java
wollok.lang.MessageNotUnderstoodException: a AsustadorNato[nivelMotivacion=100, edad=25, puntosTerrorInnatos=200] does not understand cantina()
    at wollok.lang.Object.messageNotUnderstood(name,parameters) [/wollok.wlk:151]
    at wollok.example.monstersinc.Monster.scare(boy) [/wollok.wlk:151]
    at  [/workspace.wpgm:23]
```

Check last line, from the program, doesn't have a method involcation.
In this case it should be something like:

```java
wollok.lang.MessageNotUnderstoodException: a AsustadorNato[nivelMotivacion=100, edad=25, puntosTerrorInnatos=200] does not understand cantina()
    at wollok.lang.Object.messageNotUnderstood(name,parameters) [/wollok.wlk:151]
    at wollok.example.monstersinc.Monster.scare(boy) [/wollok.wlk:151]
    at wollok.lang.Program.execute() [/workspace.wpgm:23]
```

**Comment by npasserini** [comment]
I do not like wollok.lang.Program.execute() because users do not know about
a Program object.

I would prefer to use the name/package of the file and something like
<program>, for example: wollok.example.monstersinc.<program>

**Comment by javierfernandes** [comment]
Yes. Just that in that case that part of the stack trace will be
inconsistent with other stack that are all method callls. That would make
difficult implementing stack hypeelinks an navigation for example.
Maybe assuming that a program creates a class inheriring from program but
still like a methof call

wollok.example.monsterinc.myprogram.execute() ??

Navigating that will take you to the program.
